### DISCUSSION TITLE
Rich editor line break issue

### DISCUSSION THREAD

**Discussion by edmgithub** [strapi/strapi#7852] [Legacy Topics]
Hi,
I cannot achieve multiple line breaks in the built in rich text editor.
And I need to press ENTER twice to achieve 1 line break, 
so it doesn't matter how many line breaks I press in the editor it always does 1 line break in the preview mode.

Could you offer a workaround?
Thank you very much.

**Comment by derrickmehaffy** [comment]
I believe, due to how markdown works, this will largely depend on the the library used to convert markdown over to HTML (if that is what you are doing)

Markdown has a very specific way it handles formatting and will automatically cut out a lot of extra lines (you can see this when working withing Github comments :stuck_out_tongue: ).

You can swap the editor in Strapi, someone in the community has created a package that does just that and you can find an example of that on the [awesome-strapi](https://github.com/strapi/awesome-strapi#community-plugins) repo.

**Reply by edmgithub** [reply]
Thank you for the useful answer (as always)
